integration-cli/docker_cli_help_test.go

```diff
@@ -43,47 +44,33 @@ func TestCmdHelpWidth(t *testing.T) {
 		home = os.Getenv("HOME")
 	}
 
-	for _, command := range []string{
-		"attach",
-		"build",
-		"commit",
-		"cp",
-		"create",
-		"diff",
-		"events",
-		"exec",
-		"export",
-		"history",
-		"images",
-		"import",
-		"info",
-		"inspect",
-		"kill",
-		"load",
-		"login",
-		"logout",
-		"logs",
-		"port",
-		"pause",
-		"ps",
-		"pull",
-		"push",
-		"rename",
-		"restart",
-		"rm",
-		"rmi",
-		"run",
-		"save",
-		"search",
-		"start",
-		"stats",
-		"stop",
-		"tag",
-		"top",
-		"unpause",
-		"version",
-		"wait",
-	} {
+	// Pull the list of commands from the "Commands:" section of docker help
+	helpCmd := exec.Command(dockerBinary, "help")
+	out, ec, err := runCommandWithOutput(helpCmd)
+	if err != nil || ec != 0 {
+		t.Fatalf("docker help should have worked\nout:%s\nec:%d", out, ec)
+	}
+	i := strings.Index(out, "Commands:")
+	if i < 0 {
+		t.Fatalf("Missing 'Commands:' in:\n%s", out)
+	}
+
+	// Grab all chars starting at "Commands:"
+	// Skip first line, its "Commands:"
+	count := 0
+	cmds := ""
+	for _, command := range strings.Split(out[i:], "\n")[1:] {
+		// Stop on blank line or non-idented line
+		if command == "" || !unicode.IsSpace(rune(command[0])) {
+			break
+		}
+
+		// Grab just the first word of each line
+		command = strings.Split(strings.TrimSpace(command), " ")[0]
+
+		count = count + 1
```

Comment:
LOL at some point as I was learning 'go' I tried to use some variant of ++ on a variable (I think it was ++var) and go didn't support it, so I stopped even trying to use it at all.  Fixed.